test/bundlex/integration_test.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Bundlex.IntegrationTest do
+  use ExUnit.Case, async: true
+
+  test "Example test project" do
+    assert {_output, 0} =
+             System.cmd("bash", ["-c", "mix compile 1>&2"], cd: "test_projects/example")
```

Comment:
Not sure if this test is good enough. It passes even when bundlex cannot load nif but have no clue how to fix it.

Reply:
create tests within the test project and do `mix test` instead of `mix compile` here

Reply:
done